### ISSUE TITLE
make test failed in Fedora Rawhide

### ISSUE THREAD

**Issue by tankf33der** [edubart/nelua-lang#122]
At the same time tests passed in VoidLinux.

```
$ make test
<SKIP>
[PASS] standard library | require
[FAIL] standard library | builtins (./spec/stdlib_spec.lua:13/14)
./spec/tools/expect.lua:39: Expected string to contains.
Passed in:

Expected:
warning: hi
stack traceback:
        [C]: in function 'error'
        /root/nelua-lang/lualib/nelua/utils/errorer.lua:16: in function 'nelua.utils.errorer.assertf'
        ./spec/tools/expect.lua:39: in function 'spec.tools.expect.contains'
        ./spec/tools/expect.lua:126: in function 'spec.tools.expect.run'
        ./spec/tools/expect.lua:199: in function 'spec.tools.expect.run_c_from_f                                                               ile'
        ./spec/stdlib_spec.lua:14: in function <./spec/stdlib_spec.lua:13>
        [C]: in function 'xpcall'
        /root/nelua-lang/lualib/nelua/thirdparty/lester.lua:238: in function 'nelua.thirdparty.lester.it'
        ./spec/stdlib_spec.lua:13: in local 'func'
        /root/nelua-lang/lualib/nelua/thirdparty/lester.lua:163: in function 'nelua.thirdparty.lester.describe'
        ./spec/stdlib_spec.lua:7: in main chunk
        [C]: in function 'require'
        spec/init.lua:18: in main chunk
        [C]: in ?
[PASS] standard library | traits
[PASS] standard library | libc
[PASS] standard library | math
<SKIP>
```

**Comment by edubart** [comment]
Just that test failed? Can you paste the test suite summary (last line)?

**Comment by tankf33der** [comment]
> 
> 
> Just that test failed? Can you paste the test suite summary (last line)?

Only this failed.
```
493 successes / 1 failures / 13.244206 seconds
```

**Comment by edubart** [comment]
Looks like the test failed to capture outputs from `stderr`, I will have to get hands on a Fedora box later to debug that. This should be a minor issue though, and probably only that test is affected, not the language.

**Comment by tankf33der** [comment]
ArchLinux and Debian SID fully passed, Solaris SPARC failed to start test suite.

**Comment by edubart** [comment]
> ArchLinux and Debian SID fully passed, Solaris SPARC failed to start test suite.

Share the error on SPARC then, I've tested Solaris x86_64 a few weeks ago, and everything was passing. Just SPARC fails?

**Comment by tankf33der** [comment]
```
$ uname  -a
SunOS landau 5.11 11.4.35.94.4 sun4v sparc sun4v
$ gmake test
./nelua-lua spec/init.lua
luainit.lua: bad binary format (integer format mismatch)./nelua-lua: spec/init.lua:1: module 'nelua.thirdparty.lester' not found:
        no field package.preload['nelua.thirdparty.lester']
        no file '/usr/local/share/lua/5.4/nelua/thirdparty/lester.lua'
        no file '/usr/local/share/lua/5.4/nelua/thirdparty/lester/init.lua'
        no file '/usr/share/lua/5.4/nelua/thirdparty/lester.lua'
        no file '/usr/share/lua/5.4/nelua/thirdparty/lester/init.lua'
        no file '/usr/local/lib/lua/5.4/nelua/thirdparty/lester.lua'
        no file '/usr/local/lib/lua/5.4/nelua/thirdparty/lester/init.lua'
        no file '/usr/lib/lua/5.4/nelua/thirdparty/lester.lua'
        no file '/usr/lib/lua/5.4/nelua/thirdparty/lester/init.lua'
        no file './nelua/thirdparty/lester.lua'
        no file './nelua/thirdparty/lester/init.lua'
        no file '/usr/local/lib/lua/5.4/nelua/thirdparty/lester.so'
        no file '/usr/local/lib/lua/5.4/loadall.so'
        no file './nelua/thirdparty/lester.so'
        no file '/usr/lib/lua/5.4/nelua/thirdparty/lester.so'
        no file '/usr/lib/lua/5.4/loadall.so'
        no file './nelua/thirdparty/lester.so'
        no file ''
        no file '/usr/local/lib/lua/5.4/nelua.so'
        no file '/usr/local/lib/lua/5.4/loadall.so'
        no file './nelua.so'
        no file '/usr/lib/lua/5.4/nelua.so'
        no file '/usr/lib/lua/5.4/loadall.so'
        no file './nelua.so'
        no file ''
stack traceback:
        [C]: in function 'require'
        spec/init.lua:1: in main chunk
        [C]: in ?
gmake: *** [Makefile:138: test] Error 1
$
```

**Comment by edubart** [comment]
> ```
> $ uname  -a
> SunOS landau 5.11 11.4.35.94.4 sun4v sparc sun4v
> $ gmake test
> ./nelua-lua spec/init.lua
> luainit.lua: bad binary format (integer format mismatch)
> ```

I've fixed that error in commit https://github.com/edubart/nelua-lang/commit/c99f3bf29a161af09024b7f1f7fde7bd513704b1, it happened because I've started to use some bytecode compiled Lua scripts, and now I've noticed that Lua bytecode is not portable across systems with different endianness, so I've removed the use of bytecode scripts, all tests are passing now for me on **Debian s390x** (which is big-endian). Thanks for reporting.

> make test failed in Fedora Rawhide

I could not reproduce the failing test on Fedora Rawhide, I've booted it's live image `Fedora-Workstation-Live-x86_64-Rawhide-20210911.n.0.iso` and all tests passed for me, I will try to actually install and update it to see if can reproduce.
